recipes/rocketpy/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "rocketpyalpha" %}
+{% set version = "0.9.2" %}
+
+package:
+  name: "rocketpy"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3e5e2e7b59edeb5b03f514b0b6957438b8d12508167d7d35faadda4383ec8c1c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - matplotlib >=3.0
+    - numpy >=1.0
+    - pip
+    - python
+    - scipy >=1.0
+    - requests
+  run:
+    - matplotlib >=3.0
```

Comment:
If you are not using any of the `pyqt` features of `matplotlib`, this should possibly be `matplotlib-base` instead of `matplotlib`:

https://github.com/conda-forge/conda-forge.github.io/issues/933

Reply:
Interesting suggestion. Agreed!

Applied the suggestion in the latest commits.